go.mod

```diff
@@ -5,14 +5,22 @@ go 1.13
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
 	github.com/banzaicloud/bank-vaults/pkg/sdk v0.3.0
+	github.com/docker/cli v0.0.0-20200130152716-5d0cf8839492
+	github.com/docker/go v1.5.1-1 // indirect
 	github.com/go-logr/logr v0.1.0
+	github.com/golangplus/testing v0.0.0-20180327235837-af21d9c3145e
+	github.com/gorilla/mux v1.7.2
 	github.com/hashicorp/vault/api v1.0.4
 	github.com/icza/gox v0.0.0-20200525134802-370c390b446f
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/operator-framework/operator-sdk v0.17.1
+	github.com/shortstories/logrusr v0.0.0-20191107062355-5bdb2130acc3
+	github.com/sirupsen/logrus v1.5.0
```

Comment:
Where do these logging dependencies come from?

Reply:
Some testing with other logr interfaces, as I was a bit fed up with json logging for local testing... Forgot to `go mod tidy`